proj.py

```diff
@@ -0,0 +1,134 @@
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+import pandas as pd
+import seaborn as sns
+import sqlite3
+
+
+CON = ""
+C = ""
+DF = ""
+
+
+def load_data():
```

Comment:
Add docstrings to your functions to tell more about the function.

https://www.python.org/dev/peps/pep-0008/#documentation-strings

Reply:
docstrings added to all functions!